test/models/notification_test.rb

```diff
@@ -12,7 +12,7 @@ class NotificationTest < ActiveSupport::TestCase
   end
 
   test 'readonly' do
-    assert_equal Set['event_id', 'user_id', 'system_name'],
+    assert_equal %w[event_id user_id system_name],
                  Notification.readonly_attributes
```

Comment:
Originally order of elements was insignificant. Now order is significant. Did `Set` stop working for whatever reason?

Reply:
I think it's related to https://github.com/rails/rails/commit/429f0a7d67ae85395e549a0d33269d1c58175a29

But basically, the test result was just very clear:

```
Minitest::Assertion: --- expected
+++ actual
@@ -1 +1 @@
-#<Set: {"event_id", "user_id", "system_name"}>
+["event_id", "user_id", "system_name"]
```

Well, this is the same order in which the attributes are defined here: https://github.com/3scale/porta/blob/7a81a0e9a91de0634c9fb347eb658e4faa377b1f/app/models/notification.rb#L20

But I'd agree if you suggested to change the assertion to `assert_same_elements`.

Reply:
Yes, it was :) 
I've changed the test to use `assert_same_elements` instead of `assert_equal`, so even if the order changes, the test should still pass.